app/src/main/res/values-b+pt+BR/strings.xml

```diff
@@ -0,0 +1,27 @@
+<resources>
+    <string name="app_name">Preço Por Unidade</string>
```

Comment:
Excellent!
It's a shame I didn't get a translation for app_name in NL (my bad).